src/widgetastic_patternfly4/ouia.py

```diff
@@ -0,0 +1,45 @@
+import importlib
+import sys
+
+from widgetastic.utils import ParametrizedLocator
+
+
+class OUIAMixin:
+
+    ROOT = ParametrizedLocator(
+        './/*[@data-ouia-component-type="PF4/{@component_type}" and '
```

Comment:
followup note: if we generalize the prefix this could be extracted for other libs to use

Reply:
I'm not sure what other libs you are talking about. If we plan to generalize this xpath for consuming it by other libs I don't think they should import wt.pf4 only because of OUIA.

Reply:
we could move the helper to wt core